iceberg/iceberg-handler/src/test/queries/positive/iceberg_rest_catalog.q

```diff
@@ -20,6 +22,7 @@
 -- Mask iceberg version
 --! qt:replace:/(\S\"iceberg-version\\\":\\\")(\w+\s\w+\s\d+\.\d+\.\d+\s\(\w+\s\w+\))(\\\")/$1#Masked#$3/
 
+set hive.stats.autogather=false;
```

Comment:
Not mandatory yet. Just curious why this does not work with autogather

Reply:
I don't remember the error message, but without stats autogather it was failing in query analyzer due to missing table and column statistics.